components/tabs/tab-mixin.js

```diff
@@ -11,9 +11,19 @@ export const TabMixin = superclass => class extends SkeletonMixin(superclass) {
 
 	static get properties() {
 		return {
+			/**
+			 * Use to select the tab
```

Comment:
Should we state something to the effect of only one `d2l-tab` should be selected? Hopefully that is obvious. 🤷‍♂️